internal/gateway/backend/testutil.go

```diff
@@ -140,10 +141,15 @@ func StartTestBackend(name string, maxLeaseTime time.Duration) (*Services, strin
 		os.Exit(3)
 	}
 
-	pool, err := receiver.StartPool(cfg.ReceiverPath, cfg.NumReceivers, cfg.MockReceiver)
+	smgr, err := stats.NewStatisticsMgr()
+	if err != nil {
+		os.Exit(5)
+	}
+
```

Comment:
We make `NewStatisticsMgr` failable, but it cannot really fail beside out of memory (I guess).

Maybe we should be a little more brave and make infallible the `NewStatisticsMgr`, if we change it and make it failable in a future, the compiler will let us know.